lib/src/barback/asset_environment.dart

```diff
@@ -175,13 +194,26 @@ class AssetEnvironment {
   ///
   /// If this is `null`, then the environment is "live" and all updates will
   /// go to barback immediately.
-  Set<AssetId> _modifiedSources;
+  final _modifiedSources = <AssetId, ChangeType>{};
+
+  /// The amount of time to wait in between file system events before running
+  /// a new build. Servers will be paused after the first event is received
```

Comment:
[Make the first sentence a stand-alone paragraph](https://www.dartlang.org/guides/language/effective-dart/documentation#do-make-the-first-sentence-a-standalone-paragraph).